crates/cadenza-eval/src/eval.rs

```diff
@@ -184,6 +187,12 @@ impl Eval for Literal {
     }
 }
 
+/// Helper to extract a Span from an Ident AST node.
+fn span_from_ident(ident: &Ident) -> Span {
```

Comment:
We should just add a `span` function on all of the AST nodes rather than defining it all in the eval crate.